extensions/interactions/MathEquationInput/protractor.js

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end testing utilities for Math Equation Input
+ * interaction
+ */
+
+var objects = require(process.cwd() + '/extensions/objects/protractor.js');
+
+var customizeInteraction = function() {
+  // There are no customizations.
+};
+
+var expectInteractionDetailsToMatch = async function(elem) {
+  expect(
+    await elem.element(by.tagName(
+      'oppia-interactive-math-equation-input')).isPresent()
```

Comment:
Could you use a `protractor-test-*` class here instead? You might have to make a new one. This helps stop frontend code changes like refactoring class names from breaking the e2e tests

Reply:
All of the other interactions have used the `oppia-interactive-{interaction-name}` tag pattern. Also, we'd have to add the `protractor-test-*` class to the div that contains the interaction. This div is created dynamically and is not hard-coded anywhere.

Reply:
Ahh, okay